src/sentry/models/monitorenvironment.py

```diff
@@ -29,6 +29,7 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_monitorenvironment"
         indexes = [models.Index(fields=["monitor", "environment"])]
```

Comment:
We could also remove this index since the unique will do the same thing

Reply:
ah...smart